### PR TITLE
Test alpha/beta features

### PR DESCRIPTION
**Pull request by pacoxu**

### CODE REVIEW

**Comment by pacoxu** [comment]
/test pull-kubernetes-e2e-kind-beta-features

- test to fix evented PLEG

File: staging/src/k8s.io/apiserver/pkg/features/kube_features.go [hunk 1]
```diff
@@ -285,7 +285,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	APIServerTracing: {Default: true, PreRelease: featuregate.Beta},
 
-	ValidatingAdmissionPolicy: {Default: false, PreRelease: featuregate.Beta},
+	ValidatingAdmissionPolicy: {Default: false, PreRelease: featuregate.Alpha},
```
Comment (pacoxu) [inline staging/src/k8s.io/apiserver/pkg/features/kube_features.go:288]: `pull-kubernetes-e2e-kind-beta-features` will enable all beta feature that is default false.

I changed here to make apiserver not enable ValidatingAdmissionPolicy FG when running `pull-kubernetes-e2e-kind-beta-features`.

**Comment by aojea** [comment]
beta fails with 

```

Kubernetes e2e suite: [It] [sig-api-machinery] ValidatingAdmissionPolicy [Privileged:ClusterAdmin] [FeatureGate:ValidatingAdmissionPolicy] [Beta] should type check a CRD expand_less | 2s
-- | --
{ failed [FAILED] DeferCleanup callback returned error: the server could not find the requested resource In [DeferCleanup (Each)] at: k8s.io/kubernetes/test/e2e/apimachinery/validatingadmissionpolicy.go:332 @ 01/10/24 11:43:03.208  There were additional failures detected after the initial failure. These are visible in the timeline


```

alpha does not run 🤔

**Pull request closed without merging**